Description: Ignore some tests
 For buildd, we need to skip some tests that fail on powerpc. Specifically :
 - src/sphgettime_t and src/sphgtod_t still fail and can be ignored for now as
 they compare the time taken with a constant value.
 - src/sphdirectpcqueue_ttt takes hours to complete.
Origin: https://github.com/sphde/sphde/commit/d404fe4bed885c18080caee7db75f48914e4c66b
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -155,11 +155,11 @@
 
 # Test programs invoked with 'make check'
 TESTS = 
-TESTS            += sphgtod_t
+#TESTS            += sphgtod_t
 sphgtod_t_SOURCES = tests/sphgtod_t.c
 sphgtod_t_LDADD   = .libs/libsphgtod.a libsphde.la
 
-TESTS            += sphgettime_t
+#TESTS            += sphgettime_t
 sphgettime_t_SOURCES = tests/sphgettime_t.c
 sphgettime_t_LDADD   = .libs/libsphgettime.a libsphde.la
 
@@ -245,7 +245,7 @@
 sphsinglepcqueue_ttt_SOURCES = tests/sphsinglepcqueue_ttt.c
 sphsinglepcqueue_ttt_LDADD   = libsphde.la
 
-TESTS                 += sphdirectpcqueue_ttt
+#TESTS                 += sphdirectpcqueue_ttt
 sphdirectpcqueue_ttt_SOURCES = tests/sphdirectpcqueue_ttt.c
 sphdirectpcqueue_ttt_LDADD   = libsphde.la
 
